_data/accessibility-tests/list.yml

```diff
@@ -0,0 +1,55 @@
+title: List
+component_status: pass
+test_items:
+# Zoom/screen magnification items
+  - summary: Text retains functionality when screen is magnified.
+    summary_additional: |
+      When you zoom into 200%,
+      you can still see all list content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.4
+# Screen reader items
+  - summary: Screen reader announces list.
+    summary_additional: |
+      When you listen to a list with a screen reader,
+      the screen reader announces that it is a list and tells you how many items are in the list.
+      For example, "List; four items."
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.6.1
+    wcag_criterion: 1.3.1
+  - summary: Screen reader announces list items in the correct order.
+    summary_additional: |
+      When you listen to a list with a screen reader,
+      the screen reader announces list items in the same order that they appear on the screen (e.g., top to bottom for English pages).
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.6.1
+    wcag_criterion: 1.3.1
+  - summary: Screen reader correctly reads unordered lists.
+    summary_additional: |
+      When you listen to an unordered list with a screen reader,
+      you hear "bullet item name" before each list item.
```

Comment:
@amycole501  @alex-hull 

**issue:** In voiceover,  I hear only "bullet" for unordered list items. I recommend we include that as an acceptable readout for this test. Let me know what you think!

**suggestion:**

```suggestion
      When you listen to an unordered list with a screen reader,
      you hear "bullet item name" or "bullet" before each list item.
```

Reply:
updated and added 'or "bullet"' to summary details for "screen reader correctly reads unordered lists"

Reply:
Resolved in [e1b0c19](https://github.com/uswds/uswds-site/pull/2746/commits/e1b0c192cb02e4647a06ee6629bcb4bf69166d30)

Reply:
I think making this change makes sense. Just to cover various screen readers.